app/lib/models.js

```diff
@@ -14,18 +14,32 @@ function undefinedValuesCheck (array) {
 
 class BaseModel {
   constructor(info) {
-
     this._info = info
-
     info.columns.map(name => {
       this[name] = info.data[name]
     })
   }
 
-  static _cleanParams(obj) {
+  update(params) {
+    return new Promise((fulfill, reject) => {
+      db(this.constructor.tableName)
+      .where(
+        this.constructor.primaryId, 
+        this[this.constructor.primaryId]
+      ).update(this._cleanParams(params))
+      .returning("*")
+      .then((objs) => {
+        // TODO? update params on this instance
+        // and not just the new one so that
+        // if it is re-used they are correct?
+        this.constructor._getSingleResponse(objs, fulfill)
+      })      
+    })
+  }
```

Comment:
so this requires that the primary Id be supplied in the params.
perhaps this is just me but would it be more explicit to have that primaryId be a second param? that way it is explicit